vortex-buffer/src/buffer.rs

```diff
@@ -270,6 +271,16 @@ impl<T> Buffer<T> {
     }
 }
 
+impl<T> Debug for Buffer<T> {
```

Comment:
would be nice to indicate that the buffer has been truncated...

Reply:
Good idea